CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- **ActionMenu** now uses `react-overlays` and is being added to the DOM in
```

Comment:
this is an implementation detail that is not supposed to be relevant for the user; if this fixes an issue, it should be under `### Fixed` and refer only to the issue at hand